doc/whats_new.rst

```diff
@@ -39,6 +39,7 @@ Bug
 API
 ~~~
 
+-  :func:`make_bids_basename`, :func:`make_bids_folders`, and :func:`make_dataset_description` are now available from `mne_bids.write`. The `mne_bids` base package now only exposes :func:`read_raw_bids` and :func:`write_raw_bids`, by `Stefan Appelhoff`_ (`#196 <https://github.com/mne-tools/mne-bids/pull/196>`_)
```

Comment:
I would keep in the main namespace the functions you use in examples and that you expect users to use. For example we agree that print_dir_tree should not be but maybe make_bids_basename, make_bids_folders or make_dataset_description could be used by most users? wdyt?

Reply:
fair enough, I'll add it to the main namespace (except print_dir_tree)